units/commit_queue.go

```diff
@@ -95,6 +104,10 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		return
 	}
 	nextItem := cq.Next()
+	if nextItem == "" {
```

Comment:
nextItem == "" for an empty queue.
this would fail anyway on line 111, but better to catch it here.